### PR TITLE
{chem}[intel/2019a] GULP v5.1

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
cc @OleHolmNielsen, in case it's (still?) useful

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/7b3c7e33c7caa15bce50b626c01ef3e4 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/1d99cbd109be0cfc500319bc7948baac for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/7a7450192a966c7235f15514a1df8478 for a full test report.

**Comment by OleHolmNielsen** [comment]
Test report by @OleHolmNielsen
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
svol.fysik.dtu.dk - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/a05a997dee138edca4c6894c2e0b6e7b for a full test report.

**Comment by OleHolmNielsen** [comment]
Test report by @OleHolmNielsen
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
svol.fysik.dtu.dk - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/8ac75560c566bef48d52e3a941f85cc3 for a full test report.

**Comment by OleHolmNielsen** [comment]
Test report by @OleHolmNielsen
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
svol.fysik.dtu.dk - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/d7441a2c3d2f57093a8d281341887b87 for a full test report.

**Comment by OleHolmNielsen** [comment]
Test report by @OleHolmNielsen
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
svol.fysik.dtu.dk - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/871feaf7d95649d6c23f03ffb00ab51c for a full test report.

**Comment by OleHolmNielsen** [comment]
Thanks a lot, GULP builds correctly on our 4 Intel hardware architectures! 
Can you kindly add GULP to the officially supported EB modules?
Thanks, Ole

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2414.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2319e9c2fd6e118925ef8809aeeab3ea for a full test report.

File: easybuild/easyconfigs/g/GULP/GULP-5.1-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+build_cmd = "./mkgulp -m -f -c intel"
+
+files_to_copy = [
+    (['gulp'], 'bin'),
+    (['../Docs', '../Examples', '../Libraries', '../Utils'], ''),
+]
+
+sanity_check_paths = {
+    'files': ['bin/gulp'],
+    'dirs': [],
```
Comment (boegel) [inline easybuild/easyconfigs/g/GULP/GULP-5.1-intel-2019a.eb]: @migueldiascosta Maybe we should also list `Docs` & co here, to ensure they're in place as expected?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar05.gastly.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/1797963028232c166c15d032bb26b8dd for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/1eab2aef8565828a35487ad75ee1fb3b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2414.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1a0e2274da3dd82cc8ba6e0e6c6916a7 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Comment by OleHolmNielsen** [comment]
Fantastic work guys, thanks a lot!

**Pull request merged**